src/20_Components/amp-selector.html

```diff
@@ -108,20 +134,17 @@
 
   <!-- ## Forms -->
   <!--
-Inside a form, `amp-selector` behaves like a radio-button/checkbox group and submits the selected value(s) against the name of the amp-selector.
+Inside a form, `amp-selector` behaves like a radio-button/checkbox group and submits the selected value(s) against the name of the amp-selector. Here we've styled an amp-selector to function like a radio list. Using `amp-selector` for this has the advantage that bindings using `amp-bind` are easier to implement.
```

Comment:
When you say 

> bindings using `amp-bind` are easier to implement

is there already a sample we could link to? Or maybe add a reference to amp-bind, otherwise it can be confusing.